spec/integration/puppet_x/puppetlabs/scheduled_task/taskscheduler2_spec.rb

```diff
@@ -146,6 +146,17 @@ def create_test_task(task_name = nil, task_compatiblity = PuppetX::PuppetLabs::S
       it 'should have the specified action arguments' do
         expect(subject.action(task_definition, 1).Arguments).to eq('/c exit 0')
       end
+
+      context 'it should behave like an idempotent resource' do
```

Comment:
I'm not sure how I feel about an idempotency test here.  This is an integration test for the puppet_x classes which are called via the provider.  Puppet and the provider determine idempotency, not this class.

There is already an idempotency check in the `should_create` acceptance test.

Reply:
Yeah, this probably belongs in provider.